File: be/src/main/java/com/example/newswebsite/entities/Article.java
package com.example.newswebsite.entities;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "article")
public class Article {
    @Id
    private String id;

    private String userId;

    private String image;

    private String title;

    private String content;

    //Chưa duyệt, Đã duyệt, Không được duyệt, Không hoạt động
    private String status;

    private String categoryId;

    private String publishedDate;

    private String updatedDate;

    private String censorId;
}

File: be-node/models/article.js
const mongoose = require('mongoose')
const Schema = mongoose.Schema;

const articleSchema = new Schema({
    id:{
        type: String,
        unique: true
    },
    userId:{
        type: String
    },
    image:{
        type: String
    },
    title:{
        type: String
    },
    content:{
        type: String
    },
    status:{
        type: String
    },
    categoryId:{
        type: String
    },
    status:{
        type: String
    },
}, {timestamps: true})

module.exports = mongoose.model('Article', articleSchema)